### PR TITLE
Added fact sharing forum and styling.

### PR DESCRIPTION
**Pull request by juliareichel**
- Set up Fact Object class
- Stored user input in datastore
- Retrieved data from the database and displayed it
- Added styling to fact tab

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/FactPost.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* Class representing a user's "post" (fun fact)*/
```
Comment (damargulis) [inline portfolio/src/main/java/com/google/sps/data/FactPost.java:1]: move this to the line just above the class definintion

File: portfolio/src/main/java/com/google/sps/data/FactPost.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  public FactPost(String message) {
+    fact = message;
+    postTime = String.valueOf(System.currentTimeMillis());
+  }
+
+  public FactPost(String message, long id, String time) {
+    fact = message;
+    postId = id;
+    postTime = String.valueOf(System.currentTimeMillis());
```
Comment (damargulis) [inline portfolio/src/main/java/com/google/sps/data/FactPost.java:19]: Shouldn't this use the passed in time?

File: portfolio/src/main/java/com/google/sps/servlets/DataStoreKeys.java [hunk 3]
```diff
@@ -2,4 +2,5 @@
 
 public class DataStoreKeys {
 	public static final String EMAIL_ENTITY = "Email";
+  public static final String FACT_ENTITY = "Fact";
```
Comment (damargulis) [inline portfolio/src/main/java/com/google/sps/servlets/DataStoreKeys.java:5]: fix indentation

File: portfolio/src/main/java/com/google/sps/servlets/FactServlet.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Google LLC
```
Comment (damargulis) [inline portfolio/src/main/java/com/google/sps/servlets/FactServlet.java:1]: Remove all this license text

File: portfolio/src/main/webapp/facts.js [hunk 5]
```diff
@@ -9,4 +9,22 @@ function addRandomFact() {
   const fact = FACTS[Math.floor(Math.random() * FACTS.length)];
   const factContainer = document.getElementById('fact_container');
   factContainer.innerText = fact;
+}
+
+function getPost() {
```
Comment (damargulis) [inline portfolio/src/main/webapp/facts.js:14]: Nice arrow functions here! This whole js section looks great!

File: portfolio/src/main/java/com/google/sps/data/FactPost.java (hunk 1 above)
Comment (juliareichel) [reply portfolio/src/main/java/com/google/sps/data/FactPost.java:1]: done!

File: portfolio/src/main/java/com/google/sps/servlets/FactServlet.java (hunk 4 above)
Comment (juliareichel) [reply portfolio/src/main/java/com/google/sps/servlets/FactServlet.java:1]: complete

File: portfolio/src/main/java/com/google/sps/data/FactPost.java (hunk 2 above)
Comment (juliareichel) [reply portfolio/src/main/java/com/google/sps/data/FactPost.java]: thanks for this catch!

**Pull request merged**